src/app/api/schemas.py

```diff
@@ -8,15 +9,35 @@ class UserIn(BaseModel):
     username: str = Field(..., min_length=3, max_length=50)
 
 
+class UserCreate(UserIn):
```

Comment:
I created `UserCreate` as input model for user creation (POST /users). For now it contains username/pwd/scopes but in the future it may contain additional fields that should not be part of `UserIn` (maybe a user group id, ...)